addons/mail/static/src/components/message/message.xml

```diff
@@ -178,7 +178,7 @@
                             </ul>
                         </t>
                     </div>
-                    <t t-if="message.subject and !message.isSubjectSimilarToOriginThreadName and threadView and threadView.thread and (threadView.thread.mass_mailing or [env.messaging.inbox, env.messaging.history].includes(threadView.thread))">
```

Comment:
To double check if they are used elsewhere in the template, but due to removing all of this, you might be able to remove some stuff from `useStore`.